test/unit/mock/mock_http_client.js

```diff
@@ -700,6 +700,114 @@ function buildRequestOutputMappings(clientInfo)
             }
         }
     },
+    {
+      request:
```

Comment:
do we use this mock definition in the new test?

Reply:
That mock request corresponds to the first request in the test where only the `sqlText` is specified: https://github.com/snowflakedb/snowflake-connector-nodejs/pull/639/files#diff-abf0655fdf782710cc811e2041916a2f5f3384c5747e42898cf9e5afb8ead03eR960